src/main/java/com/radixdlt/identifiers/RadixAddress.java

```diff
@@ -30,7 +30,7 @@
 /**
  * base58 address based on a public key
  */
-public final class RadixAddress {
+public final class RadixAddress implements Magical {
```

Comment:
If we are going to go with the interface, then getMagic() needs to have an @Override annotation, which your IDE should have warned you about.